modules/strongdm/roles.tf

```diff
@@ -0,0 +1,32 @@
+locals {
+  roles = toset(["db-user", "ssh-user", "kubernetes-user"])
+}
+
+resource "sdm_role" "all_access" {
+  count     = local.create_roles ? 1 : 0
+  name      = "all-access"
```

Comment:
`all-access` - is this name required by strongDM?
or should we use the `label` module to create unique names?

Reply:
I'm unsure if this is required but worth looking into. A cursory google search does not seem like it's a required name.